server/db/models/index.js

```diff
@@ -1,12 +1,33 @@
 const User = require('./user')
 const Product = require('./product')
 const Category = require('./category')
+const Order = require('./order')
+const OrderProduct = require('./OrderProduct')
```

Comment:
Ben note:  Change this name to LineItem--be more descriptive and denotes there is business logic there.

Reply:
I will make this change over now before merging.